packages/govuk-frontend/src/govuk/common/configuration/govuk-frontend-component-configurable.jsdom.test.mjs

```diff
@@ -13,6 +16,46 @@ describe('GOVUKFrontendComponentConfigurable', () => {
     document.body.classList.add('govuk-frontend-supported')
   })
 
+  describe('executes configOverride function', () => {
+    it('defined by child class', () => {
```

Comment:
As far as I understand the CharacterCount case (and I'll admit I'm treading water), the use case is when there's, say, a `maxlength` in the config passed the component, but a `maxwords` on the dataset and we want to ensure that the `maxlength` property gets tossed and the `maxwords` gets added, right?

This test covers an attribute that has the same name getting correct precedence - do we test for a case where we have two conflicting properties and we want only one of them? (Maybe we do this on the CharacterCount component tests?)

Reply:
Yeah, that's right about CharacterCount! I think maybe I could simply this test to just checking if the correct function (the one provided by child) is being called, since the logic of the function arguably doesn't matter here (that's what is being tested by the CharacterCount test which does cover this I think).

Reply:
Could we shift this test to the `set configuration on initialisation to` section and name it something like:

`it('dataset overrides if configOverride set', () => {`

Then here we could just do the check on the function call.

Reply:
Moved this!